tensorflow/lite/kernels/internal/tensor.h

```diff
@@ -52,12 +52,13 @@ class VectorOfTensors {
     int num_tensors = tensor_list.size;
```

Comment:
Suggestion: Add a `Update` function to update `VectorOfTensors`. It should work just like the constructor.